stream_alert/rule_processor/rules_engine.py

```diff
@@ -403,3 +418,36 @@ def process(cls, input_payload):
                     alerts.append(alert)
 
         return alerts
+
+    @classmethod
+    def get_intelligence(cls, intel_dir='threat_intel'):
```

Comment:
I'd suggest renaming this method from being prefixed with `get_` to being something like `load_intelligence` or `populate_intelligence`, since `get_..` implies it will return something